File: client/registration/src/app/shared/domain/score.ts
// private Long id;
// private Double score;
// private String comment;
export class Score{
    id: number;
    score: number;
    comment: string;
    constructor(arg?:{
        id: number,
        score: number,
        comment: string,
    }){

        this.comment = arg.comment;
        this.score = arg.score;
        this.id = arg.id;
    }
}

File: server/src/main/java/com/shnu/part/domain/Score.java
package com.shnu.part.domain;

import org.springframework.web.bind.annotation.GetMapping;

import javax.persistence.*;

@Entity
public class Score {
    @Id
    @GeneratedValue
    private Long id;
    private Double score;
    private String comment;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Double getScore() {
        return score;
    }

    public void setScore(Double score) {
        this.score = score;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

}
